src/popup/components/TransferSendForm.vue

```diff
@@ -62,6 +63,41 @@
       </template>
     </InputAmount>
 
+    <template v-if="showPayloadForm">
+      <div
+        v-if="!(formModel.payload && formModel.payload.length)"
+        class="payload-add-wrapper"
+      >
+        <div
+          class="payload-btn"
+          @click="editPayload"
+        >
+          <PayloadAddIcon class="payload-add-icon" />
+          {{$t('modals.send.payload') }}
+        </div>
+        <QuestionIcon
+          class="icon-btn"
+          @click="openPayloadInformation"
+        />
+      </div>
+
+      <Payload
+        v-else
+        :payload="formModel.payload"
+        class="payload"
+      >
+        <div class="payload-options">
+          <EditIcon
+            class="icon-btn"
+            @click="editPayload"
```

Comment:
The icon itself is not clickable (not accessible by the keyboard for example). I suggest to wrap the icon with additional interactive element and add the click event onto it.